packages/new-polymath-ui/src/components/inputs/Checkbox/styles.ts

```diff
@@ -34,6 +39,7 @@ export const CheckboxInput = styled.label`
   height: 1.125rem;
   background-color: #fff;
   user-select: none;
+  vertical-align: top;
```

Comment:
I don't think this has any effect as the parent is `flex` and vertically centered.

Reply:
Agreed